definitions/divorce/json/CaseEvent/CaseEvent-nonprod.json

```diff
@@ -41,22 +41,49 @@
     "Name": "Welsh Translation Resolved",
     "Description": "Welsh Translation Resolved",
     "DisplayOrder": 2,
-    "PreConditionState(s)": "BOTranslationRequested;WelshResponseAwaitingReview",
+    "PreConditionState(s)": "BOTranslationRequested;WelshResponseAwaitingReview;WelshDNisRefused",
     "PostConditionState": "*",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/welsh-state-intercept",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/welsh-event-intercept",
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "16/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "boWelshGrantDnMakeDecision",
+    "Name": "Make a Decision auto",
+    "Description": "Make a Decision",
+    "DisplayOrder": 3,
+    "PreConditionState(s)": "BOTranslationRequested",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/dn-about-to-be-granted",
+    "RetriesTimeoutURLAboutToSubmitEvent": "30,30",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/dn-decision-made",
+    "RetriesTimeoutURLSubmittedEvent": "30,30",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "AddWelshRefusalAdditionalInfo",
+    "Name": "Welsh Refusal Additional Info",
+    "Description": "Welsh Refusal Additional Info",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "BOTranslationRequested",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public"
+  },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
-    "ID": "aosSubmittedUndefended",
-    "Name": "AOS Received (undefended)",
+    "ID": "boWelshAosSubmittedUndefended",
+    "Name": "AOS Received (undefended) auto",
```

Comment:
remove the 'auto' from the name to make it the same as the original ie. "AOS Received (undefended)"